lib/update-fetch.js

```diff
@@ -163,10 +163,13 @@ exportables.downloadTgz = function(tgzUrl, fileMap) {
       return resolve(files);
     });
 
-
     remote.ifReachable(remote.BUILDS_HOSTNAME).then(() => {
+
       var req = request.get(tgzUrl);
 
+      var startTimeout = () => setTimeout(() => {reject("what should happen on timeout reached");}, 10000);
+      var timeout = startTimeout();
```

Comment:
Two minor fixes: 

- Travis is telling us that you need to use single quotes ;)
- Reject with: 
  ```js
  reject(new Error('The attempt to download a build has timed out. Check your network connection and try again.'));
  ```